docs/examples/image_morphology/morphological_operations.jl

```diff
@@ -0,0 +1,73 @@
+# ---
+# title: Convex Hull, Filling, Thinning, ClearBorder
+# author: Ashwani Rathee
+# date: 2021-7-4
+# ---
+
+# In this demonstration, we will be covering morphological operations like
+# convexhull, image thinning, hole filling and border clearning.
+
+using Images, Noise, ImageMorphology
+using HistogramThresholding
+using MosaicViews, ImageDraw
+using TestImages
+
+img_src = testimage("morphology_test_512")
+
+# Thresholding of image
+
+edges1, counts1 = build_histogram(Gray.(img_src), 256)
+t1 = find_threshold(UnimodalRosin(), counts1[1:end], edges1)
+img_input = zeros(Bool, axes(Gray.(img_src)))
+for i in CartesianIndices(img_src)
+    img_input[i] = Gray.(img_src[i]) < t1 ? 0 : 1
+end
+img_input
+
+# This is the image that we will be using throughout this demonstration
+# in its boolean form as most of operations are applied on binary images. 
+
+Gray.(img_input)
+
+# ## Convex Hull
+
+# Convex Hull operation outer most cover-like boundary of a binary image and 
+# returns the vertices of convex hull as a CartesianIndex array.
+
+cordinates = convexhull(img_input)
+push!(cordinates, cordinates[1])
+img_convex_hull = zeros(RGB, axes(Gray.(img_src)))
+draw!(img_convex_hull, Path(cordinates), RGB(1))
+img_convex_hull
```

Comment:
Maybe we can draw the convex hull on top of the original image?

Reply:
sure thing 👍🏻